app/app-maintenance.js

```diff
@@ -1,28 +1,19 @@
 const express = require('express')
-const nunjucks = require('nunjucks')
+const nunjucksSetup = require('./services/nunjucks-setup')
 const path = require('path')
 const favicon = require('serve-favicon')
 const onFinished = require('on-finished')
 const log = require('./services/log')
 
 const app = express()
 const serviceName = 'Help with Prison Visits'
+const organisationName = 'HMPPS'
 const developmentMode = app.get('env') === 'development'
 
-const appViews = [
-  path.join(__dirname, '../node_modules/govuk_template_jinja/'),
-  path.join(__dirname, '../node_modules/govuk-frontend/'),
-  path.join(__dirname, 'views')
-]
+const applicationRoles = require('./constants/application-roles-enum')
```

Comment:
Completely forgot but I think you can remove the roles stuff from this file as it's just a maintenance page. Just realised it doesn't include the routes file, just the include on line 62. Might be worth running locally in maintenance mode just to check the header when no-one is logged in, i.e. name etc. Sorry about that, tis a silly app this one 😂